src/api-service/SocketSessionUtils.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2022 IROHA LAB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { DatabaseService } from '../services/DatabaseService';
+
+export async function verifySession(payload: any, dbService: DatabaseService): Promise<boolean> {
```

Comment:
Is there a reason dbService is not injected but passed in?

Reply:
Injecting an injectable service needs to be created as a class and decorated with `@injectable()`  and initialized by the Inversify container. this is only a utility method. I want to keep it simple.